components/expenses/Expense.tsx

```diff
@@ -236,6 +237,10 @@ function Expense(props) {
   const isEditing = status === PAGE_STATUS.EDIT || status === PAGE_STATUS.EDIT_SUMMARY;
   const isEditingExistingExpense = isEditing && expense !== undefined;
   const inDrawer = Boolean(drawerActionsContainer);
+  const availableTypes = [commentTypes.COMMENT];
+  if (LoggedInUser?.isHostAdmin(data.expense.account)) {
+    availableTypes.push(commentTypes.PRIVATE_NOTE);
+  }
```

Comment:
A bit sad to have this check; all other expense permissions are handled by the API through `expense.permissions`, which has a lot of benefits. Would be great to make it consistent.

Reply:
Good point. I'll implement this properly since we shifted from the "context" concept.